go-controller/pkg/ovn/policy.go

```diff
@@ -328,11 +328,11 @@ func getMulticastACLMatch() string {
 }
 
 func getMulticastACLIgrMatchV4(addrSetName string) string {
-	return "ip4.src == $" + addrSetName + " && ip4.mcast"
+	return "(ip4.src == $" + addrSetName + " && ip4.mcast)"
 }
 
 func getMulticastACLIgrMatchV6(addrSetName string) string {
-	return "ip6.src == $" + addrSetName + " && " + ipv6DynamicMulticastMatch
+	return "(ip6.src == $" + addrSetName + " && " + ipv6DynamicMulticastMatch + ")"
```

Comment:
just double checking that your syntax is like you want it? line 331 looks slightly different than this one.

Reply:
yeah, this worked on my dual-stack KIND setup